core/src/com/unciv/ui/worldscreen/AlertPopup.kt

```diff
@@ -63,6 +63,15 @@ class AlertPopup(val worldScreen: WorldScreen, val popupAlert: PopupAlert): Popu
                     close()
                 })
             }
+            AlertType.BorderConflict -> {
+                val translatedNation = worldScreen.gameInfo.getCivilization(popupAlert.value).getTranslatedNation()
+                addLeaderName(translatedNation)
+                addGoodSizedLabel("Remove your troops in our border immediately!").row()
+                val responseTable = Table()
+                responseTable.add(getCloseButton("Sorry."))
+                responseTable.add(getCloseButton("Never!"))
```

Comment:
This should decrease influence, and add a 'refused to remove troops' countdown - so this doesn't pop up every single turn

Reply:
In original civ you also have this mechanism